common/importer/chrome_importer_utils_win.cc

```diff
@@ -20,6 +20,17 @@ base::FilePath GetChromeUserDataFolder() {
   return result;
 }
 
+base::FilePath GetVivaldiUserDataFolder() {
```

Comment:
why do we have different method names for all of these? Same with having a separate class https://github.com/brave/brave-core/pull/13990/files#diff-42126d1d544d75e6aa2195c7612d48f62a665cac430f2e4cf0aa37a923e59253R11 ?
Why can't we just have a generic class for chromium browsers that takes the user data dir as a param in the constructor?

Reply:
removed Vivaldi and Edge importers as discussed